work_queue/src/work_queue_worker.c

```diff
@@ -869,17 +872,18 @@ static int do_task( struct link *master, int taskid, time_t stoptime )
 			}
 			free(env);
 		} else if(!strcmp(line,"end")) {
-			work_queue_process_compute_disk_needed(p);
+			//work_queue_process_compute_disk_needed(p);
```

Comment:
Our process, p, is not yet defined because we want our task to have all the necessary information (such as disk if specified). This is so the process can in turn use the information from the task to determine if it should use disk_alloc or create_dir.